engine/engine-sources.gypi

```diff
@@ -23,17 +25,20 @@
 			'src/redraw.h',
 			'src/region.h',
 			'src/resolution.h',
+			'src/skiatypeface.h',
 			'src/textlayout.h',
 			'src/tilecache.h',
 			'src/bitmapeffect.cpp',
 			'src/bitmapeffectblur.cpp',
 			'src/color.cpp',
 			'src/combiners.cpp',
+			'src/customfont.cpp',
 			'src/customprinter.cpp',
 			'src/font.cpp',
 			'src/fonttable.cpp',
 			'src/glcontext.cpp',
 			'src/glcontext.h',
+			'src/freetype-font.cpp',
```

Comment:
@livecodepanos could you place this before the glcontext lines to preserve alphabetical order